CMakeLists.txt

```diff
@@ -156,16 +156,17 @@ SET(XNNPACK_OPERATOR_SRCS
 SET(XNNPACK_COLD_SRCS ${XNNPACK_OPERATOR_SRCS})
 LIST(APPEND XNNPACK_COLD_SRCS
   src/init.c
+  src/memory.c
```

Comment:
Not sure you want to categorize this as hot or cold.

Reply:
Cold LGTM. Only the code that runs inside `xnn_run_operator` (e.g. thread pool entry points) is considered hot.